lib/proxy_api/template.rb

```diff
@@ -11,12 +11,12 @@ def template_url
         return response["templateServer"]
       end
     rescue SocketError, Timeout::Error, Errno::EINVAL, Errno::ECONNRESET,
```

Comment:
I don't think I like catching the exception at all.  It looks like we would silently ignore the problem?  I don't see where an error is generated, only logging.